src/interpreter/plugin/FinancialPlugin.ts

```diff
@@ -211,6 +212,45 @@ export class FinancialPlugin extends FunctionPlugin {
         {argumentType: ArgumentTypes.NUMBER, greaterThan: 0},
       ]
     },
+    'FVSCHEDULE': {
+      method: 'fvschedule',
+      parameters: [
+        {argumentType: ArgumentTypes.NUMBER},
+        {argumentType: ArgumentTypes.RANGE},
+      ]
+    },
+    'NPV': {
+      method: 'npv',
+      parameters: [
+        {argumentType: ArgumentTypes.NUMBER},
+        {argumentType: ArgumentTypes.ANY},
+      ],
+      repeatLastArgs: 1
+    },
+    'MIRR': {
+      method: 'mirr',
+      parameters: [
+        {argumentType: ArgumentTypes.RANGE},
+        {argumentType: ArgumentTypes.NUMBER, greaterThan: -1},
```

Comment:
Why do we require that? Actually everything is possible, only for -1 we return DIV0

Reply:
ok, but then why does =MIRR({3,4,-3},-2,-2) results in NUM?